proposals/blogProposal.md

```diff
@@ -0,0 +1,11 @@
+The Web of Things standardization activities are now visible(?) through our new animation video that we have been working on for the last couple of months. You can see how any web developer can implement Internet of Things solutions while being interoperable and not locked into frameworks. Additionally, the W3C WoT allows the experience of the developers to be transferred to different application domains such as agriculture, building or industrial automation, smart home and more!
```

Comment:
I am not sure with visible either. Alternatives that come to my mind: `described`, `available` or simply saying `are now on a new animation...`

Reply:
How about "illustrated"?

Reply:
"Explained" would be another possible alternative to "visible"